ops/app/oci-images.nix

```diff
@@ -0,0 +1,24 @@
+let
+  inherit (inputs) std self;
+
+  # TODO: express this as OCI labels (what they are for)
+  BUILD_INFO = builtins.toJSON {
```

Comment:
This is consumed by the application which is why I believe that we can not use labels for it

Also, please make the variable name camelCase

Reply:
Oh, I just realized that since we embed build info into OCI it will get rebuilt each time, even if the code has not changed

Reply:
@rhyslbw did you backlog this? Removing this and adopting best practices through using OCI labels probably needs some changes to how we think of today's `/meta`.